Database/Terraform/outputs.tf

```diff
@@ -0,0 +1,22 @@
+# This file outputs data from the database
+# The output data is useful for connecting to the database
+
+data "aws_db_instance" "database" {
+  db_instance_identifier = "c11-hermes-db"
+}
```

Comment:
Is this part necessary? Could be instead directly access resources from `main.tf`?

Reply:
True this part wasn't needed, thanks